superset/dashboards/api.py

```diff
@@ -196,13 +198,14 @@ def post(self) -> Response:
         """
         if not request.is_json:
             return self.response_400(message="Request is not JSON")
-        item = self.add_model_schema.load(request.json)
+        try:
+            item = self.add_model_schema.load(request.json)
         # This validates custom Schema with custom validations
-        if item.errors:
-            return self.response_400(message=item.errors)
+        except ValidationError as err:
+            return self.response_400(message=err.messages)
```

Comment:
Being picky, but is it more pythonic to assign `error` as opposed to `err`? (Applies to other places, too)